modules/geospatial/src/test/java/org/apache/ignite/internal/processors/query/h2/H2IndexingAbstractGeoSelfTest.java

```diff
@@ -629,11 +663,11 @@ public Enemy(int campId, String name) {
     protected static class EnemyCamp implements Serializable {
         /** */
         @QuerySqlField(index = true)
-        Geometry coords;
+        final Geometry coords;
```

Comment:
protected?

Reply:
not, it used in `checkDistributedQuery()` by direct reference.